src/main/java/br/ufpe/cin/mergers/handlers/renaming/KeepBothMethodsRenamingHandler.java

```diff
@@ -11,6 +11,15 @@
 import de.ovgu.cide.fstgen.ast.FSTNode;
 import de.ovgu.cide.fstgen.ast.FSTTerminal;
 
+/**
+ * Alternative implementation of the renaming handler.
+ * 
+ * This handler always keeps both methods involved in the renaming or deletion,
```

Comment:
"This handler always resolve a renaming or deletion conflict by keeping the involved declarations, even if they would lead to a conflict if merged"

correto @jvcoutinho ?

Reply:
Isso.